invenio_app_ils/series/mappings/v6/series/series-v1.0.0.json

```diff
@@ -273,6 +273,20 @@
             }
           },
           "type": "object"
+        },
+        "electronic_volumes_description": {
+          "type": "text"
+        },
+        "physical_volumes": {
+          "properties": {
+            "description": {
+              "type": "text"
+            },
+            "location": {
+              "type": "keyword"
```

Comment:
Since it is not a controlled vocabulary, maybe better to have it as `text`?
Just thinking that I might put as value `Building 10 A` and for another `Building 10 B`, maybe I want to search for `Building 10` and find both...
WDYT?